tests/pkg_jsmn/simple.c

```diff
@@ -53,6 +52,13 @@ static int jsoneq(const char *json, jsmntok_t *tok, const char *s)
     return -1;
 }
 
+static void print_string(char *prefix, const char *buf, size_t len)
+{
+    strncpy(printbuf, buf, len);
```

Comment:
Maybe add `len = MIN(len, sizeof(printbuf) - 1);` before this line? This should not overflow here anyway, but maybe a paranoid safety measure can prevent some long debugging session if the app gets extended.

Reply:
Good idea, I added that change and directly amended it to the last the commit.